docs/install-cicero.md

```diff
@@ -0,0 +1,53 @@
+# Install Cicero
+
+## How to install Cicero locally
+
+### Clone repository locally
+
+```
+git clone https://github.com/input-output-hk/cicero.git
+```
+
+### Fix for Local paths
+
+For local paths to work in Cicero it's necessary to add the following line to nixos/configs/vm.nix
+Afterwards the Qemu VM and Cicero need to be restarted.
+
+```
+   nix = {
+     package = pkgs.nix_2_5;
+	 ...
+   };
+```
+
+### Provide github token with read:org, repo flags
+
+```
+cat >> ~/.netrc << EOF
+machine github.com
+login github_username
+password github_token
+
+machine api.github.com
+login github_username
+password github_token
+EOF
+```
+
+### Enter the Cicero development shell
+```
+nix develop
+```
+
+### Start a development Qemu VM with Nomad, nomad-follower, Vault and Spongix:
+```
+nixos-shell --flake .
+```
+
+### Open another terminal and run Cicero outside the VM:
+```
+nix develop
```

Comment:
I really like this change ❤️ 
Currently [How to run in README](https://github.com/input-output-hk/cicero/blob/cic-81/README.md#how-to-run) is a bit out of sync with mdBook. Perhaps we could replace this section of readme with this file.

Same situation is with https://github.com/input-output-hk/cicero/tree/cic-81#prerequisites and install-nix.md

Reply:
Replaced now by:
How To Run
[Install Nix](https://cicero-handbook.infra.aws.iohkdev.io/install-nix.html) section of the Cicero Handbook.
[Install Cicero](https://cicero-handbook.infra.aws.iohkdev.io/install-cicero.html) section of the Cicero Handbook.